pom.xml

```diff
@@ -35,9 +35,18 @@
         <module>sample-service</module>
         <module>practitioner-service</module>
         <module>trial-config-service</module>
+        <module>sc-config-server</module>
     </modules>
 
     <dependencies>
+        <dependency>
```

Comment:
Do you know what are these dependencies? Do we need them at the parent?

Reply:
yeah I think that the spring-cloud dependencies should be at MTS-Service level